src/WattleScript.Interpreter/Tree/Statements/ForLoopStatement.cs

```diff
@@ -37,7 +37,7 @@ public ForLoopStatement(ScriptLoadingContext lcontext, Token nameToken, Token fo
 			}
 			else
 			{
-				m_Step = new LiteralExpression(lcontext, DynValue.NewNumber(1));
+				m_Step = new LiteralExpression(lcontext, DynValue.One);
```

Comment:
Remove

Reply:
Fixed in 4c95935184ba919fceacf7c9de78d2078e4f9abf